src/applications/simple-forms/21-0972/pages/preparerQualifications1.js

```diff
@@ -1,22 +1,88 @@
-import { preparerQualificationsQuestionTitle } from '../config/helpers';
+import {
+  preparerQualificationsQuestionTitle,
+  preparerQualificationsQuestionLabels,
+} from '../config/helpers';
 import GroupCheckboxWidget from '../../shared/components/GroupCheckboxWidget';
-import { preparerQualificationsOptions } from '../definitions/constants';
+import { claimantIdentificationDisplayOptions } from '../definitions/constants';
 
 /** @type {PageSchema} */
+const commonUiSchema = {
+  preparerQualifications: {
+    'ui:widget': GroupCheckboxWidget,
+    'ui:errorMessages': {
+      required:
+        'You must select at least one relationship, so we can process your certification.',
+    },
+    // different titles between all the variants
+    'ui:options': {
+      forceDivWrapper: true,
+      // different labels between all the variants
+      showFieldLabel: true,
+    },
+  },
+};
 export default {
-  uiSchema: {
+  uiSchemaA: {
```

Comment:
can you keep all of the uiSchema and schema without a postfix? the intellisense that /** @type {PageSchema} */ helps with (when/if loaded) expects uiSchema and schema explicitly in order to work.

Reply:
let's sync on this topic offline